e2e/cephfs_helper.go

```diff
@@ -93,27 +128,27 @@ func createCephfsStorageClass(
 
 	// fetch and set fsID from the cluster if not set in params
 	if _, found := params["clusterID"]; !found {
-		var fsID string
-		fsID, err = getClusterID(f)
+		fsID, err := getClusterID(f)
 		if err != nil {
-			return fmt.Errorf("failed to get clusterID: %w", err)
+			return nil, fmt.Errorf("failed to get clusterID: %w", err)
 		}
 		sc.Parameters["clusterID"] = fsID
 	}
 
-	timeout := time.Duration(deployTimeout) * time.Minute
+	return sc, nil
+}
 
+func createStorageClass(c kubernetes.Interface, sc *scv1.StorageClass) error {
+	timeout := time.Duration(deployTimeout) * time.Minute
 	return wait.PollUntilContextTimeout(context.TODO(), poll, timeout, true, func(ctx context.Context) (bool, error) {
-		_, err = c.StorageV1().StorageClasses().Create(ctx, &sc, metav1.CreateOptions{})
+		_, err := c.StorageV1().StorageClasses().Create(ctx, sc, metav1.CreateOptions{})
 		if err != nil {
 			framework.Logf("error creating StorageClass %q: %v", sc.Name, err)
 			if isRetryableAPIError(err) {
 				return false, nil
 			}
-
```

Comment:
Here, an in other places, you removed an empty line before `return`. The coding style in Ceph-CSI (and **golangci-lint**?) expects these. Please add them back.